src/Commands/Upstream/UpdatesApplyCommand.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * @file
+ * Contains Pantheon\Terminus\Commands\Upstream\UpdatesApplyCommand
+ */
+
+
+namespace Pantheon\Terminus\Commands\Upstream;
+
+
+use Terminus\Collections\Sites;
+use Terminus\Exceptions\TerminusException;
+use Terminus\Models\Site;
+
+class UpdatesApplyCommand extends UpstreamCommand
+{
+
+  /**
+   * Applies the available upstream updates to the given site.
+   *
+   * @authorized
+   *
+   * @name upstream:updates:apply
+   * @aliases upstream-updates upstream:updates
+   *
+   * @param string $environment Name of the environment to retrieve
+   *
+   * @return void
+   *
+   * @field-labels
+   *   hash: Commit ID
+   *   datetime: Timestamp
+   *   message: Message
+   *   author: Author
+   *
+   * @throws TerminusException
+   *
+   * @usage terminus upstream:updates:list <site-name>
+   *   Lists the available updates for the site called <site-name>
+   */
+  public function applyUpstreamUpdates($environment, $options = ['updatedb' => true, 'accept-upstream' => true]) {
+      $parts = explode('.', $environment);
+      $site_id = $parts[0];
+      $env = !empty($parts[1]) ? $parts[1] : 'dev';
+      if (in_array($env, ['test', 'live',])) {
+         throw new TerminusException(
+              'Upstream updates cannot be applied to the {env} environment',
+              compact('env')
+          );
+      }
+
+      $updates = $this->getUpstreamUpdates($site_id);
+      $count = count($updates);
+      if ($count) {
+          $this->log()->notice(
+              'Applying {count} upstream update(s) to the {env} environment of {site_id}...',
+              compact('count', 'env', 'site_id')
+          );
+
+          $site  = $this->getSite($site_id);
+
+          $workflow = $site->applyUpstreamUpdates(
+              $env,
+              $options['updatedb'],
+              $options['accept-upstream']
+          );
+          $workflow->wait();
+          // $this->workflowOutput($workflow);
```

Comment:
@ronan The messages that Terminus 0.x extracts from workflows to display needs to be copied into the Terminus 1.x commands. If we were implementing i18n now, we could put all the text into a dictionary but that's been kiboshed for 1.0. Do not output anything (to STDOUT) which the user did not explicitly ask for (as in info commands). Please output the messages via `$this->log()->notice()`.